lib/pages/signup/about-page.js

```diff
@@ -9,6 +9,26 @@ export default class AboutPage extends BaseContainer {
 	}
 
 	submitForm() {
-		return driverHelper.clickWhenClickable( this.driver, By.css( '.about__submit-wrapper button.is-primary' ), this.explicitWaitMS );
+		return driverHelper.clickWhenClickable( this.driver, By.css( '.about__submit-wrapper button.is-primary' ) );
+	}
+
+	enterSiteDetails( siteTile, siteTopic, { showcase = false, share = false, sell = false, educate = false, promote = false } = {} ) {
```

Comment:
Small typo in this and the next line: `SiteTile` -> `SiteTitle`.

Reply:
Thanks 👍